### ISSUE TITLE
Can we set learning rate 0.0001 at beginning

### ISSUE THREAD

**Issue by gokhanli** [pythonlessons/Keras-ResNet-tutorial#4]
i want start algrotihm with different "Learning Rate" value.
For example 0.0001 or 0.1 but it is start with 0.001 learning rate value
